### ISSUE TITLE
unclear license (GPL or MIT license or both)

### ISSUE THREAD

**Issue by 0noketa** [sparkfun/SparkFun_HyperDisplay#9]
### unclear license
* different description on README.md and License.md .　GPL version 3 on README, MIT License on License.md.

**Comment by westinpigott** [comment]
Agree.  It seems likely that it is MIT given that seems to be standard for SparkFun, but the ambiguity needs to be clarified.
